### ISSUE TITLE
Error loading toolchain on OS X with bazel 0.4.4

### ISSUE THREAD

**Issue by sayrer** [pubref/rules_node#6]
When I try to use a node_library rule, I get this error:

ERROR: /Users/sayre/vsco/go/src/github.com/vsco/godel/optional/js/BUILD:3:1: every rule of type node_library implicitly depends upon the target '@org_pubref_rules_node_toolchain//:node_tool', but this target could not be found because of: no such package '@org_pubref_rules_node_toolchain//': Not a file: /private/var/tmp/_bazel_sayre/4de71d28335d84ef3a9372b4f1cd1773/external/nodejs_darwin_amd64/BUILD.
ERROR: /Users/sayre/vsco/go/src/github.com/vsco/godel/optional/js/BUILD:3:1: every rule of type node_library implicitly depends upon the target '@org_pubref_rules_node_toolchain//:npm_tool', but this target could not be found because of: no such package '@org_pubref_rules_node_toolchain//': Not a file: /private/var/tmp/_bazel_sayre/4de71d28335d84ef3a9372b4f1cd1773/external/nodejs_darwin_amd64/BUILD.

---

In the WORKSPACE, I have:

git_repository(
    name = "org_pubref_rules_node",
    commit = "2b9d10213759d87ff6a8b543ca5a770dcfa2b4e1",
    remote = "https://github.com/pubref/rules_node.git",
)
load("@org_pubref_rules_node//node:rules.bzl", "node_repositories")
node_repositories()

**Comment by pcj** [comment]
What version of bazel are you using?

**Comment by sayrer** [comment]
LT-AT0380:godel sayre$ bazel version
Build label: 0.4.4-homebrew
Build target: bazel-out/local-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Feb 2 01:05:15 2017 (1485997515)
Build timestamp: 1485997515
Build timestamp as int: 1485997515

**Comment by pcj** [comment]
Something different about bazel 0.4.4 that's breaking it.  Will investigate.

**Comment by pcj** [comment]
Think I see the problem:  external workspaces in `$(bazel info output_base)/external` used to generate at least 2 files: `WORKSPACE`, and `BUILD`.  

Looks like in bazel 0.4.4 it's no longer `BUILD`, but `BUILD.bazel`.

**Comment by pcj** [comment]
Try this commit: c5b0347fc10822bd8f1c2fe6a3a77367621b0849

```python
git_repository(
    name = "org_pubref_rules_node",
    remote = "https://github.com/pubref/rules_node.git",
    commit = "c5b0347fc10822bd8f1c2fe6a3a77367621b0849",
)
```

**Comment by pcj** [comment]
See https://github.com/bazelbuild/bazel/issues/2492

**Comment by sayrer** [comment]
I think it is still broken. It is no longer failing because it can't find a file, but it is still showing an error.

Before:
```
ERROR: /Users/sayre/vsco/go/src/github.com/vsco/godel/optional/js/BUILD:3:1: every rule of type node_library implicitly depends upon the target '@org_pubref_rules_node_toolchain//:npm_tool', but this target could not be found because of: no such package '@org_pubref_rules_node_toolchain//': Not a file: /private/var/tmp/_bazel_sayre/4de71d28335d84ef3a9372b4f1cd1773/external/nodejs_darwin_amd64/BUILD.
```

After:

```
ERROR: /Users/sayre/vsco/go/src/github.com/vsco/godel/optional/js/BUILD:3:1: every rule of type node_library implicitly depends upon the target '@org_pubref_rules_node_toolchain//:npm_tool', but this target could not be found because of: no such target '@org_pubref_rules_node_toolchain//:npm_tool': target 'npm_tool' not declared in package '' defined by /private/var/tmp/_bazel_sayre/4de71d28335d84ef3a9372b4f1cd1773/external/org_pubref_rules_node_toolchain/BUILD.bazel.
```

**Comment by pcj** [comment]
I pushed a new commit that passed travis for versions 0.3.1 - 0.4.4, merged it and tagged it as `v0.3.3`.  If that one works please go ahead and close this issue.

**Comment by sayrer** [comment]
thanks, fixed!
